package.json

```
@@ -115,18 +115,19 @@
     "react-i18next": "11.18.6",
     "react-icons": "4.11.0",
     "react-lottie": "1.2.3",
+    "react-markdown": "^10.1.0",
```

please pin this version

```suggestion
    "react-markdown": "10.1.0",
```